react/week-1/lesson.md

```diff
@@ -110,43 +86,29 @@ JSX is a simple syntax *sugar* that looks like HTML, but is actually converted t
 
 Using JSX ([interactive version](https://jsbin.com/gabujocuda/edit?html,output)):
 
-```html
-<!DOCTYPE html>
-<html>
-<body>
-<script src="https://unpkg.com/react@16.0.0/umd/react.development.js"></script>
-<script src="https://unpkg.com/react-dom@16.0.0/umd/react-dom.development.js"></script>
-<!-- Add Babel script, which compiles JSX to React.createElement -->
-<script src="https://unpkg.com/babel-standalone@6.26.0/babel.js"></script>
-
-<div id="root"></div>
-
-<script type="text/babel">
-  const element = <div>Hello World</div>;
-
-  const rootElement = document.querySelector('#root');
-  ReactDOM.render(element, rootElement);
-</script>
+```js
+const element = <div>Hello World</div>;
 
-</body>
-</html>
+const rootElement = document.querySelector("#root");
+ReactDOM.render(element, rootElement);
 ```
 
 As you can see, this is much easier to read than both the straight `React.createElement` API and the vanilla JS API. Most people using React use JSX to write their components.
 
 > **Exercise B**
 > Change the [JSX example from above](http://jsbin.com/gekahexige/edit?html,output) to instead render a `h1` tag with the text "Hello Code Your Future".
 
-## Let's Create A React App
+## Let's create a React app
 
 The Facebook team behind React have created a tool to help you create and set up React projects. It is called [Create React App](https://github.com/facebook/create-react-app). It sets up files like we saw in the previous example, so that you don't have to.
 
 > **Exercise C**: Install & set up a Create React App by following the steps below
 
-```
-npm install -g create-react-app
+1. Open the command line and navigate to where you usually save projects
+2. Run the code below:
 
-create-react-app pokedex
+```
+npx create-react-app pokedex
```

Comment:
Just a note for the use of `npx`. This is available for npm@5.2.0 or higher. Given that the current version of npm is 6+ now, I think this should be safe to assume. But, I thought I'd note this just incase someone has an issue. 👍

Reply:
Good point. I'll check what we know about people's setup 👍

Reply:
This has already been updated on master: https://syllabus.codeyourfuture.io/react/week-1/lesson.html#lets-create-a-react-app. For simplicity of pre-installing, it has been moved to the wiki: https://docs.codeyourfuture.io/students/guides/creating-a-react-app.

Re: versioning, we made sure the students installed a recent version of Node (v13.x) when they started JS1. It comes bundled with npm 5+